src/Serializer/Tests/AbstractItemNormalizerTest.php

```diff
@@ -90,6 +90,7 @@ public function testSupportNormalizationAndSupportDenormalization(): void
             [],
             null,
             null,
+            null,
```

Comment:
all the null added argument should be optional we can remove them right?

Reply:
Complete file reverted in https://github.com/api-platform/core/pull/5758/commits/828ecb60ffeab01c34c9b6f922b208832d3a9d3d